app/views/shared/_ie_fallbacks.html.slim

```diff
@@ -0,0 +1,5 @@
+- nonce = content_security_policy_script_nonce
+
+<script nonce="#{nonce}" src='https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js'></script>
```

Comment:
Actually looking again, I do still see errors on the page, one related to Object.keys (as ES5 method). So the html5 shiv *might* be doing something, but its not resolving our JS errors.